compiler/modules/nback/init.bal

```diff
@@ -0,0 +1,24 @@
+import wso2/nballerina.bir;
+import wso2/nballerina.print.llvm;
+
+public function buildInitModule(bir:ModuleId entryModId, map<bir:FunctionSignature> publicFuncs, llvm:Context llContext) returns llvm:Module|BuildError {
+    llvm:Module llMod = llContext.createModule();
+    llvm:Builder builder = llContext.createBuilder();
+    // SUBSET support cases with no/different entry function
+    string userMainName = "main";
+    check buildMain(entryModId, userMainName, publicFuncs.get(userMainName), llMod, builder);
```

Comment:
front-end isn't requiring main: it just requires that if main exists, then it has to have a particular signature and be public. Not sure we want to require main for E test cases.  A Ballerina program doesn't have to have a main (think something that just starts a server). So I think for now our _bal_main should just not call anything if there is no user main. Maybe we can add an error/warning down the road.